src/main/java/com/example/security/JwtUtil.java

```diff
@@ -0,0 +1,61 @@
+package com.example.security;
+
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jws;
+import io.jsonwebtoken.JwtException;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.security.Keys;
+import java.nio.charset.StandardCharsets;
+import java.security.Key;
+import java.util.Date;
+import java.util.function.Function;
+import org.springframework.stereotype.Component;
+
+@Component
+public class JwtUtil {
+    private final Key secret;
+
+    //    @Value("${jwt.expiration}")
+    private final Long expiration = 3000L;
```

Comment:
Did you use `@Value("${jwt.expiration}")` with final field? 
Let's discuss it in chat

Reply:
Nope, not final, like this:
   @Value("${jwt.expiration}")
        private long expiration;

Reply:
fixed after adding jwt properties to test application.propertioes